### DISCUSSION TITLE
MONARC does not send mails, despite postfix installed and working

### DISCUSSION THREAD

**Discussion by sgl-mnrch** [monarc-project/MonarcAppFO#482] [Q&A] [answered]
I have MONARC installed on DEBIAN 11, FO only. 
Its working but it does not send mails when creating a new user. 

Shouldn't there be an invitation when a new user is created since I did not create a password for him.

Did I miss a step in the configuration?

**Comment by ruslanbaidan** [comment, marked as answer]
There is no invitation, but we can consider adding the feature in the future.
He/she has to use the reset password option.

**Comment by sgl-mnrch** [comment]
Ah, thx for the clarification. Will just invite the users then manually.
